app/views/recipes/shopping_list.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="container width-50">
+  <h2>Shopping List</h2>
+  <div class="d-flex justify-content-between">
+    <div>
+      <p>Total number of items: <%= @total_items %></p>
+      <p>Total price: <%= number_to_currency(@total_price) %></p>
+    </div>
+    <p>Recipe: <%= @recipe.name %></p>
```

Comment:
There seems to be an issue with this line here. Please do fix this error and ask for a re-review.

![image](https://user-images.githubusercontent.com/116313214/233217428-73888c0a-86f1-4b1c-8420-c883ced249c2.png)